cmd/abapLandscapePortalUpdateAddOnProduct_test.go

```diff
@@ -0,0 +1,816 @@
+package cmd
```

Comment:
The tests take a lot of time. Possibly through the wait statements. Could you rework it in a way that the wait times are reduced in a test?